src/lib/prisma.ts

```
@@ -1,14 +1,15 @@
-  prisma = new PrismaClient();
-} else {
-  if (!global.prisma) {
-    global.prisma = new PrismaClient();
-  }
-  prisma = global.prisma;
+const prismaClientSingleton = () => {
+  return new PrismaClient()
 }
 
-export default prisma;
+declare const globalThis: {
```

Biome

[error] Do not shadow the global "globalThis" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)